apps/pipelines/nodes/nodes.py

```diff
@@ -380,6 +384,7 @@ def _process(self, input, state: PipelineState) -> PipelineState:
             node_id=self.node_id,
             output=result.output,
             output_message_metadata=history_manager.output_message_metadata,
+            tag=f"{self.name}:{self.tag}" if self.tag and self.tag.strip() else None,
```

Comment:
I don't think these tags should have the node name appended to them. I think it should just be the plain value provided by the user.